node/nodeTransactions.go

```diff
@@ -60,19 +60,19 @@ func (n *Node) optionallyGetTransactionFromPool(hash []byte) (*transaction.ApiTr
 func (n *Node) getFullHistoryTransaction(hash []byte) (*transaction.ApiTransactionResult, error) {
 	miniblockMetadata, err := n.historyRepository.GetMiniblockMetadataByTxHash(hash)
 	if err != nil {
-		return nil, ErrTransactionNotFound
+		return nil, NewErrTransactionNotFound(err)
 	}
 
 	txBytes, txType, found := n.getTxBytesFromStorageByEpoch(hash, miniblockMetadata.Epoch)
 	if !found {
 		log.Warn("getFullHistoryTransaction(): unexpected condition, cannot find transaction in storage")
-		return nil, ErrCannotRetrieveTransaction
+		return nil, NewErrCannotRetrieveTransaction(err)
```

Comment:
error wrapping can happen directly here.

Reply:
Let's use the wrappers defined in `errors.go`, please (a bit cleaner).

Reply:
Fixed.